docs/development.md

```diff
@@ -355,21 +355,28 @@ To run Conformance locally, set `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE
-| `SKIP_TESTS`                | Skip running Kubernetes E2E tests.                                                                             |
+| Variable                    | Description                                                                                                   |
+|-----------------------------|---------------------------------------------------------------------------------------------------------------|
+| `AZURE_SSH_PUBLIC_KEY_FILE` | Use your own SSH key.                                                                                         |
+| `SKIP_CREATE_CLUSTER`       | Skip cluster creation.                                                                                        |
+| `SKIP_UPSTREAM_E2E_TESTS`   | Skip running upstream Kubernetes E2E tests.                                                                   |
 | `SKIP_CLEANUP`              | Skip deleting the cluster after the tests finish running.                                                      |
-| `KUBECONFIG`                | Provide your existing cluster kubeconfig filepath. If no kubeconfig is provided, `./kubeconfig` will be used.  |
-| `SKIP`                      | Regexp for test cases to skip.                                                                                 |
-| `FOCUS`                     | Regexp for which test cases to run.                                                                            |
+| `KUBECONFIG`                | Provide your existing cluster kubeconfig filepath. If no kubeconfig is provided, `./kubeconfig` will be used.     |
+| `SKIP`                      | Regexp for test cases to skip.                                                                                |
+| `FOCUS`                     | Regexp for which test cases to run.                                                                           |
 | `PARALLEL`                  | Skip serial tests and set --ginkgo-parallel.                                                                  |
-| `USE_CI_ARTIFACTS`          | Use a CI version of Kubernetes, ie. not a released version (eg. `v1.19.0-alpha.1.426+0926c9c47677e9`)          |
-| `CI_VERSION`                | Provide a custom CI version of Kubernetes. By default, the latest master commit will be used.                  |
+| `USE_CI_ARTIFACTS`          | Use a CI version of Kubernetes, ie. not a released version (eg. `v1.19.0-alpha.1.426+0926c9c47677e9`)         |
+| `CI_VERSION`                | Provide a custom CI version of Kubernetes. By default, the latest master commit will be used.                 |
 
 You can also customize the configuration of the CAPZ cluster (assuming that `SKIP_CREATE_CLUSTER` is not set). See [Customizing the cluster deployment](#customizing-the-cluster-deployment) for more details.
 
+In addition to upstream E2E, you can append custom commands to `./scripts/ci-entrypoint.sh` to run E2E from other projects against a CAPZ cluster:
```

Comment:
you might also want to edit https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/617/files#diff-1bf07101845de6da98f1c948b82c5ce0R353 which still refers to `ci-conformance.sh`?